<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToDoList</class>
 <widget class="QWidget" name="ToDoList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>607</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToDoList</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="4" rowspan="2">
      <widget class="QPushButton" name="pushButton_Cancelar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>CANCELAR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_estado">
       <item>
        <widget class="QLabel" name="label_estado">
         <property name="text">
          <string>Estado:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_estado">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_estado">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="5">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>Fecha y Hora</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label_titulo">
       <property name="text">
        <string>Lista de Tareas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPlainTextEdit" name="plainTextEdit_descripcion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>430</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QLabel" name="label_descripcion">
       <property name="text">
        <string>Descripcion:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4" rowspan="2">
      <widget class="QPushButton" name="pushButton_AgregarTarea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>72</height>
        </size>
       </property>
       <property name="text">
        <string>AGREGAR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_nombreTarea">
       <item>
        <widget class="QLabel" name="label_nombreTarea">
         <property name="text">
          <string>Tarea:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_nombreTarea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_nombreTarea">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="ListaDeTareas">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="TareasEnProceso">
      <attribute name="title">
       <string>Realizandose</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QScrollArea" name="AreaTareasEnProceso">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="AllNewTareasEnProceso">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>383</width>
            <height>337</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <spacer name="verticalSpacerTareasEnProceso">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>316</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TareasEnEspera">
      <attribute name="title">
       <string>En Espera</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="AreaTareasEnEspera">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="AllNewTareasEnEspera">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>383</width>
            <height>337</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacerTareasEnEspera">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TareasFinalizadas">
      <attribute name="title">
       <string>Finalizadas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QScrollArea" name="AreaTareasFinalizadas">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="AllNewTareasFinalizadas">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>383</width>
            <height>337</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <spacer name="verticalSpacerTareasFinalizadas">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>316</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TareasCanceladas">
      <attribute name="title">
       <string>Canceladas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QScrollArea" name="AreaTareasCanceladas">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="AllNewTareasCanceladas">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>383</width>
            <height>337</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <spacer name="verticalSpacerTareasCanceladas">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>316</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
